Index: kopete-4.11.80/protocols/CMakeLists.txt
===================================================================
--- kopete-4.11.80.orig/protocols/CMakeLists.txt	2013-11-14 16:09:23.449330409 +0100
+++ kopete-4.11.80/protocols/CMakeLists.txt	2013-11-14 16:09:23.441330409 +0100
@@ -39,8 +39,8 @@
 endif(OPENSSL_FOUND OR WIN32)
 
 if(MEDIASTREAMER_FOUND AND LIBORTP_FOUND AND WITH_WLM_MEDIASTREAMER)
-  message(STATUS "Building Windows Live Messenger voice clip support")
-  set(BUILD_WLM_MEDIASTREAMER TRUE)
+  message(STATUS "Not building Windows Live Messenger voice clip support")
+  set(BUILD_WLM_MEDIASTREAMER FALSE)
 else(MEDIASTREAMER_FOUND AND LIBORTP_FOUND AND WITH_WLM_MEDIASTREAMER)
   message(STATUS "Not building Windows Live Messenger voice clip support")
   set(BUILD_WLM_MEDIASTREAMER FALSE)
